plugins/wasm_bpf/func-load-bpf-object.cpp

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+#include "func-load-bpf-object.h"
+
+namespace WasmEdge {
+namespace Host {
+
+Expect<handle_t> LoadBpfObject::body(const Runtime::CallingFrame &Frame,
+                                     uint32_t obj_buf, uint32_t obj_buf_sz) {
+  auto *memory = Frame.getMemoryByIndex(0);
+  if (unlikely(!memory)) {
+    return Unexpect(ErrCode::Value::HostFuncError);
+  }
+  char *const object_buffer = memory->getPointer<char *>(obj_buf, obj_buf_sz);
+  if (!object_buffer) {
+    return Unexpect(ErrCode::Value::HostFuncError);
+  }
+  auto program = std::make_unique<wasm_bpf_program>();
+  int32_t res = program->load_bpf_object(object_buffer, (size_t)obj_buf_sz);
```

Comment:
Use a cast-like statement instead of ()casting.

Reply:
Fixed now